packages/nodes-base/nodes/Twist/Twist.node.ts

```diff
@@ -201,6 +228,193 @@ export class Twist implements INodeType {
+							}
+
+							attachments.push(await twistApiRequest.call(
+								this,
+								'POST',
+								`/attachments/upload`,
+								{},
+								{},
+								{
+									formData: {
+										file_name: {
+											value: Buffer.from(binaryData.data, BINARY_ENCODING),
+											options: {
+												filename: binaryData.fileName,
+											},
+										},
+										attachment_id: uuid(),
+									},
+								},
+							));
+						}
+
+						body.attachments = attachments;
+					}
+
+					if (body.direct_mentions) {
+						const direcMentions: string[] = [];
+						for (const directMention of body.direct_mentions as number[]) {
+							direcMentions.push(`[name](twist-mention://${directMention})`);
+						}
+						body.content = `${direcMentions.join(' ')} ${body.content}`;
+					}
+
+					responseData = await twistApiRequest.call(this, 'POST', '/comments/add', body);
+				}
+				//https://developer.twist.com/v3/#remove-comment
+				if (operation === 'remove') {
+					const commentId = this.getNodeParameter('commentId', i) as string;
+					qs.id = commentId;
+
+					responseData = await twistApiRequest.call(this, 'POST', '/comments/remove', {}, qs);
+				}
+				//https://developer.twist.com/v3/#get-comment
+				if (operation === 'get') {
+					const commentId = this.getNodeParameter('commentId', i) as string;
+					qs.id = commentId;
+
+					responseData = await twistApiRequest.call(this, 'GET', '/comments/getone', {}, qs);
+				}
+				//https://developer.twist.com/v3/#get-all-comments
+				if (operation === 'getAll') {
+					const workspaceId = this.getNodeParameter('workspaceId', i) as string;
+					const threadId = this.getNodeParameter('threadId', i) as string;
+					const returnAll = this.getNodeParameter('returnAll', i) as boolean;
+					const filters = this.getNodeParameter('filters', i) as IDataObject;
+					qs.workspace_id = workspaceId;
+					qs.thread_id = threadId;
+
+					Object.assign(qs, filters);
+					if (!returnAll) {
+						qs.limit = this.getNodeParameter('limit', i) as number;
+					}
+					if(qs.older_than_ts){
+						qs.older_than_ts = moment(qs.older_than_ts as string).unix();
+					}
+					if(qs.newer_than_ts){
+						qs.newer_than_ts = moment(qs.newer_than_ts as string).unix();
+					}
+
+					responseData = await twistApiRequest.call(this, 'GET', '/comments/get', {}, qs);
+				}
+				//https://developer.twist.com/v3/#update-comment
+				if (operation === 'update') {
+					const commentId = this.getNodeParameter('commentId', i) as string;
+					const updateFields = this.getNodeParameter('updateFields', i) as IDataObject;
+					const body: IDataObject = {
+						id: commentId,
+					};
+					Object.assign(body, updateFields);
+
+					if (body.actionsUi) {
+						const actions = (body.actionsUi as IDataObject).actionValues as IDataObject[];
+
+						if (actions) {
+							body.actions = actions;
+							delete body.actionsUi;
+						}
+					}
+
+					if (body.binaryProperties) {
```

Comment:
Not required, but it is always possible to define a reusable handler in `GenericFunctions.ts` to avoid duplication.